File: June/ex33.py
#!/usr/bin/python
## exercise 33

i = 0
l = int(raw_input("Enter the loop times: "))
step = int(raw_input("Enter the step: "))
numbers = []

while i < l:
	 print "At the top i is %d" %i
	 numbers.append(i)

	 i = i + step
	 print "Numbers now: ", numbers
	 print "At the bottom i is %d" %i

print "The numbers: "

for num in numbers:
	print num

xlist= [1,2,3,4]
y = int(raw_input(">"))
if y not in xlist:
	print "Damn it!"

File: Learn_Ruby_The_Hard_Way/ex33.rb

def foo_while(arg)
	numbers = []
	for i in (0..arg)
		puts "At the top i is #{i}"
		numbers.push(i)
	
		puts "Numbers now: #{numbers}"
		puts "At the bottom i is #{i}"
	end
	return numbers
end

puts "The numbers: "

for num in foo_while(6)
	puts num
end
